pyproject.toml

```diff
@@ -17,15 +17,15 @@ dependencies = [
     "nslsii",
     "pyepics",
     "aioca",
-    "pydantic<2.0",
+    "pydantic<2.0",      # Pinned by our required fastapi version, leave pinned until can check incompatibility
     "stomp-py",
     "aiohttp",
     "PyYAML",
-    "click<8.1.4",
+    "click",
     "fastapi[all]<0.99", # Later versions use a newer openapi schema, which is incompatible with swagger see https://github.com/swagger-api/swagger-codegen/issues/10446
     "uvicorn",
     "requests",
-    "dls-bluesky-core",  #requires ophyd-async
+    "dls-bluesky-core",
```

Comment:
nit: isn't that deprecated soon? shouldn't it have a comment like pydantic does?

Reply:
It is deprecated and will be removed soon, the comment just isn't relevant anymore as the dependency on ophyd-async (also) comes from dodal

Reply:
ok but a new comment wouldn't hurt